### PR TITLE
Running tests on mono

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -74,12 +76,30 @@ Task("Compile")
...
+        Configuration = configuration,
+        Verbose = false
+      });
+    }
+    else 
+    {
+      DotNetCoreBuild(project.GetDirectory().FullPath, new DotNetCoreBuildSettings {
+        Configuration = configuration,
+        Verbose = false,
+        Runtime = "unix-x64"
```
Comment (devlead) [inline build.cake]: will there be greater variation in the future.
`Runtime = IsRunningOnWindows() ? null : "unix-x64"` would have same effect,
What do you think?

Comment (jchannon) [inline build.cake]: Yup fair point

File: build.cake [hunk 2]
```diff
@@ -74,12 +76,30 @@ Task("Compile")
   .Does(() =>
 {
   var projects = GetFiles("./**/*.xproj");
+
+  if (!IsRunningOnWindows())
```
Comment (devlead) [inline build.cake]: `IsRunningOnUnix()​` probaly more clear here.

Comment (jchannon) [inline build.cake]: Didnt know that was a thing

**Comment by jchannon** [comment]
@devlead tweaks applied

**Comment by devlead** [comment]
@jchannon LGTM :+1:

**Pull request merged**